src/MySql/custom/Update-AzMySqlFlexibleServer.ps1

```diff
@@ -181,6 +188,40 @@ function Update-AzMySqlFlexibleServer {
                 $null = $PSBoundParameters.Remove('StorageAutogrow')
             }
 
+            if ($PSBoundParameters.ContainsKey('MaintenanceWindow')) {
```

Comment:
please ensure c# codes is going to handle the case when MaintenanceWindow is absent.

Reply:
I saw no problem updating a server with parameters other than MaintenanceWindow.